### PR TITLE
vertical setting

### PR DESCRIPTION
**Pull request by ethayer**

### CODE REVIEW

**Comment by ArturT** [comment]
Could you update readme?

File: spec/avatarly_spec.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'pry'
```
Comment (ArturT) [inline spec/avatarly_spec.rb]: I guess you don't need this here anymore :)

**Comment by ArturT** [comment]
@ethayer Thanks for PR. I've just released avatarly v1.5.0.

**Comment by ethayer** [comment]
Ah, sorry for the messy PR.  I think I meant to do the PR into the master of my fork before ultimately going for the PR.  Lots of messing around first.

Thanks for the merge.  This fixes some issues I have with a non-center baseline font.

**Pull request merged**